pcs/common/reports/messages.py

```diff
@@ -6858,6 +6858,19 @@ def message(self) -> str:
         return "cannot determine local site ip, please specify site parameter"
 
 
+@dataclass(frozen=True)
+class BoothSiteIpNotSpecifiedOnArbitrator(ReportItemMessage):
+    """
+    booth site must be specified when granting tickets from an arbitrator.
+    """
+
+    _code = codes.BOOTH_SITE_IP_NOT_SPECIFIED_ON_ARBITRATOR
+
+    @property
+    def message(self) -> str:
+        return "Unable to operate booth ticket on arbitator node without specifying site ip"
+
+
```

Comment:
No need for a new report message. There is existing report message `BoothCannotDetermineLocalSiteIp` which does not mention arbitrator node and can be used for other cases when there is no cib available on a node.